app/src/main/java/org/mozilla/fenix/home/sessioncontrol/SessionControlUIView.kt

```diff
@@ -25,6 +26,8 @@ private fun SessionControlState.toAdapterList(): List<AdapterItem> {
         tabs.reversed().map(AdapterItem::TabItem).forEach { items.add(it) }
         if (mode == Mode.Private) {
             items.add(AdapterItem.DeleteTabs)
+        } else if (BuildConfig.COLLECTIONS_ENABLED) {
```

Comment:
This requires a rebuild and re-install to flip users between collections_enabled and collections_disabled. Should our feature flag be a Fretboard experiment instead with a reasonable default if the user hasn't grabbed experiments yet?

Reply:
@colintheshots I wasn't planning on A/B testing it. I just wanted a way to be able to land code in chunks until it's done and not block the beta if we don't get it done in time.